docs/digital-identity.md

```diff
@@ -151,9 +158,10 @@ example above).
 
 ##### Check Sentry for `CapturedSensitiveException` events
 
-Try the debugging steps described above under "email address out of
-date": if there is another user record with the user's email address,
-creating their account will fail.
+Try the [debugging steps](#check-sentry-for-capturedsensitiveexception-events)
```

Comment:
~~Hmmm this anchor link references the section we're already in?~~

Oh wait, we have two identical headings! In which case, I'd be tempted to put a `<a name="debugging-steps"></a>` in the appropriate place and then link to `#debugging-steps` here.

Reply:
Thanks, I've fixed this and also made some small tweaks.